lib/Parser.js

```diff
@@ -234,8 +234,8 @@ Parser.prototype.initializeEvaluating = function() {
 	this.plugin("evaluate MemberExpression", function(expression) {
 		var expr = expression;
 		var exprName = [];
-		while(expr.type === "MemberExpression" && !expr.computed) {
-			exprName.unshift(expr.property.name);
+		while(expr.type === "MemberExpression") {
+			exprName.unshift(expr.property.name || expr.property.value);
```

Comment:
computed is a property of a MemberExpression, which decides between `a.b` and `a[b]`.

http://astexplorer.net/#/2rUJe2TD3H